packages/orbit-components/src/Stack/Stack.stories.tsx

```diff
@@ -206,7 +206,7 @@ export const TabletProperties = () => {
   };
 
   return (
-    <Stack tablet={tablet}>
+    <Stack tablet={tablet} spaceAfter={spaceAfter}>
```

Comment:
This does not belong here

Reply:
It's just had unused spaceAfter knob ;)

Reply:
It was before the `spaceAfter` was added to the `tablet` object defined above, as it was missing and we added a few days ago 😄 so we can (and should) remove it from here, as all other media query stories don't feature it